base/ltcmd.dtx

```diff
@@ -3681,18 +3681,38 @@
 % \end{macro}
 %
 % \begin{macro}{\@@_grab_v_aux_put:N}
+% \changes{v1.2d}{2024/03/21}{Collect \cs{endlinechar} as \cs{obeyedline}}
 %   Storing one token in the collected argument. Most tokens are
 %   converted to category code $12$, with the exception of active
 %   characters, and spaces (not sure what should be done for those).
 %    \begin{macrocode}
+%<latexrelease>\IncludeInRelease{2024/11/01}{\@@_grab_v_aux_put:N}%
```

Comment:
you plan this for the fall release? not sure it is that it needs that much of a lead time, but ok.

Reply:
Editing error, I'll edit for June